templates/layouts/tag.html

```diff
@@ -1,63 +0,0 @@
----
-layout: default
-tagging: blog
-paginate: true
----
-{@io.quarkiverse.roq.frontmatter.runtime.model.NormalPage page}
-<div class="blog-page grid-wrapper">
```

Comment:
Hi @jasondlee , in the page preview on your account, the tags stopped working, if you click on the tag (all blogs contain some tags) then the 404 gets returned.

Reply:
Pffft. I'll look into that...